pkg/cluster/internal/providers/docker/network.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package docker
 
 import (
-	"crypto/sha1"
```

Comment:
This is not actually a cyptographic hash and sha256 is overkill.

Reply:
I don't want to change the default CIDR to satisfy some overzealous linter.

Reply:
let's just drop gosec instead